components/article/client/view.coffee

```diff
@@ -184,23 +187,39 @@ module.exports = class ArticleView extends Backbone.View
       imagesLoaded $(this), =>
         $(this).width $(this).children('img').width()
 
-  fillwidth: (el, cb) ->
-    if @$(el).length < 1 or $(window).width() < 700
+  fillwidth: (el, cb=->) ->
+    if @$(el).length < 1 or $(window).width() < 400
       @$(el).parent().removeClass('is-loading')
       return cb()
     $list = @$(el)
+    if $(window).height() > 700
+      newHeight = $(window).height() * .8
     $list.fillwidthLite
       gutterSize: 30
+      targetHeight: newHeight || 600
       apply: (img) ->
         img.$el.closest('li').width(img.width)
       done: (imgs) ->
         # Make sure the captions line up in case rounding off skewed things
         tallest = _.max _.map imgs, (img) -> img.height
         $list.find('.artwork-item-image-container').each -> $(this).height tallest
+        # Account for screens that are both wide & short
+        listWidth = _.map imgs, (img) -> img.width
+        parentWidth = _.reduce(listWidth, (a, b) ->
+                return a + b
+              , 0) + (($($list).children().length - 1) * 30)
+        if $($list).width() > parentWidth
+          $($list).css({'display':'flex', 'justify-content':'center'})
```

Comment:
This method is starting to get a little confusing to read -- might be helpful to extract parts (like you mentioned to me earlier) or leave more descriptive comments, especially around the areas with specific numbers like the `parentWidth` calculation and the different calculations for each of the breakpoints. But overall great job at thinking about each edge case and being careful with this!